functions/src/notifications/populateBillNotificationEvents.ts

```diff
@@ -37,7 +37,7 @@ export const populateBillNotificationEvents = functions.firestore
 
         billCourt: court,
         billId: newData?.id,
-        billName: newData?.content.Title,
+        billName: newData?.id,
```

Comment:
OOC, what purpose does changing `billName` serve if it just contains the same data we already have in `billId`?